src/components/heading/heading.scss

```diff
@@ -61,6 +61,17 @@ $max-width-permalink-shift: math.div($min-width-permalink-shift * 16 - 1, 16);
   }
 }
 
+/**
+ * Weight modifier
+ *
+ * The default font weight for headings is set to 700. This modifier will
+ * change that to a lighter weight of 300.
+ */
+
+.c-heading--weight-light {
```

Comment:
Do you think the word `weight` is necessary? Would `c-heading--light` be confusing?

Reply:
I'll change it.

Reply:
I added `weight` just in case we wanted to add more in the future. I'll remove it though